package.json

```diff
@@ -46,6 +46,7 @@
     "estraverse": "^4.2.0",
     "esutils": "^2.0.2",
     "file-entry-cache": "^2.0.0",
+    "functional-red-black-tree": "^1.0.1",
```

Comment:
I decided to unmark this as WIP because I looked for more balanced BST libraries and didn't find any good alternatives at the moment. I also tried implementing one myself for awhile, but balanced BSTs are relatively complicated and difficult to get right, so it's probably not worth adding an implementation to the repo if this module is working for us right now.